src/Evolution/Executables/ForceFree/EvolveForceFree.hpp

```diff
@@ -219,7 +222,10 @@ struct EvolutionMetavars {
           domain::Tags::Coordinates<3, Frame::ElementLogical>>,
 
       Initialization::Actions::AddComputeTags<
-          tmpl::list<ForceFree::Tags::ComputeTildeJ>>,
+          tmpl::list<ForceFree::Tags::TildeESquaredCompute,
+                     ForceFree::Tags::TildeBSquaredCompute,
+                     ForceFree::Tags::TildeEDotTildeBCompute,
```

Comment:
Maybe just inline the dot product where you need it? Remember that compute tags need memory on every element. So we usually compute intermediate quantities like this in temporary storage and discard it when the time step is done.

Reply:
@nilsvu If memory is not an issue for this evolution system (which is the case for the GH that have >100 variables to store), I'd like to have this as a tag since this term is computed fairly frequently during the time step. It's used for both flux and sources.

Reply:
But I don't have a strong opinion on this.